openradar/publishing.py

```diff
@@ -99,9 +99,11 @@ def publications(self, cascade=False):
         combinations = utils.get_product_combinations(
             datetimes=datetimes,
             prodcodes=self.prodcodes,
-            timeframes=self.timeframes,
+            timeframes=self.timeframes
         )
         for combination in combinations:
+            if self.nowcast:
+                yield products.NowcastProduct(**combination)
```

Comment:
It seems tricky to me to use the existing and obsolete NowcastProduct class for something that is no more than a decluttered composite on an extended grid. Better make a new class for it.